docs/other/install-gwms-frontend.md

```diff
@@ -1,124 +1,89 @@
+software. It **does not** cover anything but the simplest possible install.
+Please consult the [Glidein WMS reference
+documentation](http://www.uscms.org/SoftwareComputing/Grid/WMS/glideinWMS/doc.prd/install.html)
+for advanced topics, including non-`root`, non-RPM-based installation.
 
 This document covers three components of the GlideinWMS a VO needs to install:
 
 -   **User Pool Collectors**: A set of `condor_collector` processes. Pilots submitted by the factory will join to one of these collectors to form a Condor pool.
 -   **User Pool Schedd**: A `condor_schedd`. Users may submit Condor vanilla universe jobs to this schedd; it will run jobs in the Condor pool formed by the **User Pool Collectors**.
 -   **Glidein Frontend**: The frontend will periodically query the **User Pool Schedd** to determine the desired number of running job slots. If necessary, it will request the factory to launch additional pilots.
 
-This guide covers installation of all three components on the same host: it is designed for small to medium VOs (see the Hardware Requirements below). Given a significant, large host, we have been able to scale the single-host install to 20,000 running jobs.
-
-![alt text](../images/simple_diagram.png)
-
-
-Release
-=======
+This guide covers installation of all three components on the same host: it is
+designed for small to medium VOs (see the Hardware Requirements below). Given a
+significant, large host, we have been able to scale the single-host install to
+20,000 running jobs.
 
-This document reflects glideinWMS v3.2.17.
-
-How to get Help?
-================
-
-To get assistance about the OSG software please use [this page](../common/help.md).
-
-For specific questions about the Frontend configuration (and how to add it in your HTCondor infrastructure) you can email the glideinWMS support <glideinwms-support@fnal.gov>
+![GlideinWMS Architecture](/images/simple_diagram.png)
 
-To request access the OSG Glidein Factory (e.g. the UCSD factory) you have to send an email to <osg-gfactory-support@physics.ucsd.edu> (see below).
 
-Requirements
-============
+Before Starting
+---------------
 
-Host and OS
------------
+Before starting the installation process, consider the following points (consulting [the Reference section below](#reference) as needed):
 
-1. A host to install the GlideinWMS Frontend (pristine node).
-2. OS is Red Hat Enterprise Linux 6, 7, and variants (see [details](../common/yum.md)). Currently most of our testing has been done on Scientific Linux 6.
-3. Root access
+-   **User IDs:** If they do not exist already, the installation will create the Linux users `apache` (UID 48), `condor`, `frontend`, and `gratia`
+-   **Network:** The VO frontend must have reliable network connectivity, be on the public internet (no NAT), and preferably with no firewalls. Each running pilot requires 5 outgoing TCP ports. Incoming TCP ports 9618 to 9660 must be open.
```

Comment:
This recommendation about no NAT and no firewalls sounds crazy. And why would pilots need 5 outgoing TCP ports from the frontend? @bbockelm @efajardo @mambelli

Reply:
@brianhlin I assume this is the case when the frontend and the Condor Collector/Negotiator are in the same host. The reason behind the no NAT is that the frontend hosts an apache web server and then you would have to do some static port forwarding. So I kinda agee with that recommendation.

The TCP ports 9618 to 9660 open sounds too much. You really need to open only 9618 and 9620 to 9660 that would be more accurate. The 5 outgoing TCP ports is in the compute node and are ephimeral ports so I would just erase that part.